### ISSUE TITLE
Generated Postman collection: wrong request path variables from example

### ISSUE THREAD

**Issue by MarcoFierimonte** [apideck-libraries/portman#296]
Hi all,

I have an issue on generated postman collection. The request path variable seems not taking the correct value from the "example" open-api section.
This is the section of the open-api file:
![image](https://user-images.githubusercontent.com/12981741/160367289-a097af2b-f81c-44d0-89e7-bcfd7a9502b3.png)

On portman-side, I am using the following configs about generated postman:
![image](https://user-images.githubusercontent.com/12981741/160367472-bc7e69f8-7a26-40f6-88a4-b43ab6f010e0.png)

The portman-config.json file have NO override section...so the variables values should be retrived from openapi file instead the result I got on generated postman file is the following:
![image](https://user-images.githubusercontent.com/12981741/160368424-f37d88f7-23fb-45b0-adfc-3eb9775b5d24.png)

Any idea about it? Is it an issue or a miss configuration on my side?

Thanks for help.
Marco F.

**Comment by thim81** [comment]
hi @MarcoFierimonte 

Your YAML indenting is a bit off. The `example` is part of the schema section.
 In your screenshot it is part of the "path" section.

![2022-03-28 at 11 37 05](https://user-images.githubusercontent.com/952446/160370590-117561aa-9db2-44d2-b7ba-7b055c9d4c94.png)

You should indent `example` a level deeper, like this:
<img width="505" alt="2022-03-28 at 11 35 39@2x" src="https://user-images.githubusercontent.com/952446/160370446-d8f3d044-371b-440a-877c-500c275308c8.png">

**Comment by MarcoFierimonte** [comment]
Hi @thim81 thanks for your reply.

Your point is good but others params are correctly generated and they are defined in the same way. For example, this is **scenarioCode** from the openapi:
![image](https://user-images.githubusercontent.com/12981741/160373584-45b653d6-c3c6-4f55-ac6c-3948d31a60ad.png)

It is defined like projectCode and in the generated postman collection the result is correct:
![image](https://user-images.githubusercontent.com/12981741/160373860-53d8321b-6833-458a-a1a3-b8455af1046f.png)

Basically for all the params in the openapi only the "projectCode" is wrong. All the others are correct :( 

Thanks again.
Marco F.

**Comment by savage-alex** [comment]
I beleive in OpenAPI the example can be at this level. Have we tried examples as well?
![image](https://user-images.githubusercontent.com/25430683/160423009-998d42b3-a60d-43fc-9e8c-10fe07160494.png)

![image](https://user-images.githubusercontent.com/25430683/160423062-731ca810-879c-4e41-91de-0ed232e7dc1f.png)

**Comment by savage-alex** [comment]
@MarcoFierimonte could you wrap the example in single quotes 'foo' as its a string. I wonder if its being missed as its being interepreted as an INT

**Comment by MarcoFierimonte** [comment]
Hi @savage-alex , thanks for your help :) 

I generated the openapi from a maven plugin called "springdoc-openapi-maven-plugin" so I cannot change manually the API.
BTW the file is correct because the most commons tools can read it without errors. This for example is the openapi from Swagger Editor Online: 
![image](https://user-images.githubusercontent.com/12981741/160553699-43b958ed-fa47-4578-9778-907fc7add883.png)

As you can see the examples values are correctly read from the file.
This is the openapi, I shared it online for better understand the problem: [https://codebeautify.org/yaml-parser-online/y2230dc8d](url)

Thanks to all for the support.

**Comment by savage-alex** [comment]
I cant get the link to work but it does look like its being seen as an INT (see the colour difference) for the purpose of testing could you change the example to have a leading character like the example below it for user name?

**Comment by MarcoFierimonte** [comment]
Hi @savage-alex ,

I updated the link, maybe now it is working.
I also made the change you suggested and it worked !! The piece of openapi that I updated as you said was converted correctly: 
![image](https://user-images.githubusercontent.com/12981741/160557857-9b69258b-9e84-4757-8baf-0badc54b8e66.png)

Is this a bug of the conversion tool or a bug of openapi specifications? 

Thanks again.

**Comment by savage-alex** [comment]
Your most welcome. 

I had done the same thing in the past as you would expect you dont have to wrap it in quotes but when run a linter against it like spectral it complains that its not a string. (Which lead me to the theory) 

I expect its related to the conversion tool. 

Are we ok to close this Issue please?

**Comment by MarcoFierimonte** [comment]
Yes I close this issue.

Thanks.
Marco
